prisma/schema.prisma

```diff
@@ -224,6 +224,7 @@ model StatefulCover {
 }
 
 enum ComicRarity {
+  None
```

Comment:
Since we added a `None` case here, I think it makes sense to remove an "optional" flag from `ComicRarity` columns in the schema. `StatefulCover` and `StatelessCover` should have `?` remove from the `rarity` column.

(perhaps) we should also add a default `ComicRarity.None` in case no rarity was provided, but this could lead to unwanted behavior. Hmm, lets skip that part